src/main/java/com/palonskiy/initializer/DbInitializer.java

```diff
@@ -0,0 +1,27 @@
+package com.palonskiy.initializer;
+
+import org.flywaydb.core.Flyway;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.ApplicationListener;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.context.event.ContextRefreshedEvent;
+import org.springframework.stereotype.Component;
+
+@PropertySource("classpath:bd.properties")
+@Component
+public class DbInitializer implements ApplicationListener<ContextRefreshedEvent> {
+
+    @Value("${db.url}")
+    private String url;
+    @Value("${db.login}")
+    private String login;
+    @Value("${db.password}")
+    private String password;
+
+
+    @Override
+    public void onApplicationEvent(ContextRefreshedEvent contextRefreshedEvent) {
+        Flyway flyway = Flyway.configure().dataSource(url, login, password).locations("classpath:db/migration").load();
```

Comment:
code formatt:
` Flyway flyway = Flyway.configure()
.datasource...
.locations...
.load()`

Reply:
hat to refactor migration code due to error i descrided in skype